x-pack/test/functional/es_archives/endpoint/alerts/api_feature/mappings.json

```diff
@@ -389,7 +389,8 @@
                   "type": "nested"
                 },
                 "file_extension": {
-                  "type": "long"
+                  "ignore_above": 1024,
```

Comment:
This was conflicting with the Endpoint package. The file_extension should be a string, not a long.